src/commonMain/kotlin/uk/gov/hmrc/calculator/utils/tapering/TaperingExtensions.kt

```diff
@@ -34,6 +36,10 @@ internal fun Double.getTaperingAmount(maximumTaperingAmount: Double): Double {
 }
 
 @JvmSynthetic
-internal fun Double.shouldApplyTapering() = this > TAPERING_THRESHOLD
+internal fun Double.isAboveHundredThousand() = this > TAPERING_THRESHOLD
```

Comment:
Would this be clearer?
```suggestion
internal fun Double.yearlyWageIsAboveHundredThousand() = this > TAPERING_THRESHOLD
```